tlscanary/modes/regression.py

```diff
@@ -181,6 +179,12 @@ def run_regression_passes(self, host_set, report_completed=None, report_overhead
             # If no errors, no need to keep scanning
             if len(current_host_set) == 0:
                 break
+            else:
```

Comment:
Having this in an `else` block after `if ... break` is somewhat pointless, but I don't mind enough to ask for a change.